emop-maven-plugin/src/main/java/edu/cornell/MonitorMojo.java

```diff
@@ -57,7 +57,7 @@ public void execute() throws MojoExecutionException {
         if (!includeNonAffected || !includeLibraries) {
             start = System.currentTimeMillis();
             // Rewrite BaseAspect.aj to ignore non-affected classes
-            generateNewBaseAspect();
+            generateNewBaseAspect(!(!includeNonAffected || !includeLibraries));
```

Comment:
Isn't this equivalent to (includeNonAffected && includeLibraries) using DeMorgan's? Would make it easier to read